src/site/content/en/reliable/codelab-reliability-register-service-worker/index.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: codelab
+title: Create a service worker using Workbox
+author: mdiblasio
+date: 2019-05-07
+description: |
+  In this section you'll learn how to register a Workbox-powered service worker.
+glitch: wiki-offline-starter
+web_lighthouse: N/A
+---
+
+In this section you'll learn how to create a
+[Workbox](https://developers.google.com/web/tools/workbox/)-powered service
+worker, which you'll use to cache assets in later sections of the project.
+
+{% Aside 'caution' %}
+Make sure to [set up Glitch and DevTools](../codelab-reliability-setup/) if you
+haven't already.
+{% endAside %}
+
+A [service worker](https://developers.google.com/web/fundamentals/primers/service-workers/)
+is a script that your browser runs in the background, separate from a web page.
+Because they're run in the background, service workers let you create features
+similar to those of a native application (like caching assets for viewing
+offline, which you'll set up later).
+
+To create your service worker, you'll use
+[Workbox](https://developers.google.com/web/tools/workbox/), a JavaScript
+library for adding offline support to web apps. To start using Workbox import
+the `workbox-sw.js` file into your app by adding
+the following line to `src/service-worker.js`:
+
+```js
+importScripts('https://storage.googleapis.com/workbox-cdn/releases/4.3.0/workbox-sw.js');
```

Comment:
FWIW, we're up to Workbox v4.3.1 now.

I'm not sure what the best approach is for keeping this version number (relatively) up to date in the many places throughout the codelab that it's used.

On DevSite, we were able to store the current version string in a template variable and inject it into pages:

https://github.com/google/WebFundamentals/blob/0772d9a3907f17fff56db1918a8d52f6efa3b788/src/content/en/tools/workbox/_shared/workbox-sw-cdn-url.html